README.md

```diff
@@ -1,2 +1,74 @@
-# iframe-datasource-javascript-sample
-This application aims to demonstrate how to embed the data source designer module into your JavaScript-based web application using an iFrame.
+# BoldBI Datasource Iframe with Javascript Embedding
```

Comment:
Data Source Iframe Embedding with JavaScript

Reply:
Updated this line